src/lib.ts

```diff
@@ -68,13 +66,15 @@ export const globalOptions: {
   noBarf: boolean;
   askedAboutWish: boolean;
   wishAnswer: boolean;
+  pillKeeperUses: number;
 } = {
   stopTurncount: null,
   ascending: false,
   saveTurns: 0,
   noBarf: false,
   askedAboutWish: false,
   wishAnswer: false,
+  pillKeeperUses: 0,
```

Comment:
I don't understand the utility of tracking this internally. Can you explain?

Reply:
I added pillkeeper to the diet, so this is to mark how many "spleen" uses of pillkeeper the diet has budgeted for. Those pill uses will come later in the run, so we have to share this between diet and embezzler.ts somehow, and I felt this was the easiest way to accomplish that.